### PR TITLE
Add Class to Pluto-Charon System.

### PR DESCRIPTION
**Pull request by Planetarianproductions**
An updated version of the Pluto-Charon system. Works on Celestia:

testplutocharondwarf

### CODE REVIEW

**Comment by Planetarianproductions** [comment]
Here is the orbit as a dwarf:
plutocharondwarforbit

**Comment by 375gnu** [comment]
@Askaniy @LukeCEL could check this PR. You are better in astronomy :)

File: data/solarsys.ssc [hunk 1]
```diff
@@ -1539,7 +1539,7 @@ AltSurface "limit of knowledge" "Sol/Neptune/Triton"
 }
 
 ReferencePoint "Pluto-Charon" "Sol"
-{
+{   Class "dwarfplanet"
```
[the remark this answers is not in the document]
Comment (Planetarianproductions) [reply data/solarsys.ssc]: Done.

**Comment by Askaniy** [comment]
We will need this change in the future, but now this parameter for _ReferencePoint_ doesn't work in 1.7.0.

**Comment by Planetarianproductions** [comment]
So this isn't ready just yet?

**Comment by 375gnu** [comment]
@Askaniy what about adding it to Pluto?

**Comment by Askaniy** [comment]
@375gnu, it's already there:
https://github.com/CelestiaProject/Celestia/blob/master/data/solarsys.ssc#L1555

I plan to transfer Pluto from the _solarsys.ssc_ and dwarf planets from the _asteroids.ssc_ to a separate _dwarfplanets.ssc_. 
Also, ArtBlos is going to write you about _ReferencePoint_ problems in about a week.

**Comment by 375gnu** [comment]
So there is not need in this change, so I'm going to close the PR.

**Pull request closed without merging**